inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/reader/file/MonitorTextFile.java

```diff
@@ -76,6 +77,7 @@ public void monitor(FileReaderOperator fileReaderOperator, TextFileReader textFi
      */
     private class MonitorEventRunnable implements Runnable {
 
+        private static final int WAIT_TIME = 30;
```

Comment:
add a configuration for the wait time pls

Reply:
Internal startup logical, don`t have to add configuration.